client/driver/logging/universal_collector.go

```diff
@@ -170,7 +157,7 @@ func (s *SyslogCollector) configureTaskDir() error {
 
 // getFreePort returns a free port ready to be listened on between upper and
 // lower bounds
-func (s *SyslogCollector) getFreePort(lowerBound uint, upperBound uint) (net.Addr, error) {
+func (s *SyslogCollector) getListener(lowerBound uint, upperBound uint) (net.Listener, error) {
```

Comment:
@dadgar No one else needs this for now, once we find another usage of this we could move this into the helper package.